day_7/projects/caeser_cipher.rb

```diff
@@ -0,0 +1,30 @@
+class CaesarCipher
+  attr_accessor :position
+  attr_reader :alphabet
+
+  def initialize(position)
+    @position = position
+    @alphabet = ('a'..'z').to_a
+  end
+
+  def encrypt(string)
+    letter_map = Hash[alphabet.zip(alphabet.rotate(position))]
+
+    string.downcase.chars.reduce("") do |new_str, char|
+      encrypted_char = letter_map[char]
+
+      if encrypted_char.nil?
+        (new_str += char).upcase
+      else
+        (new_str += encrypted_char).upcase
+      end
+    end
+  end
```

Comment:
This is a really elegant solution with advanced use of enumerable methods `zip` and `reduce`, excellent work.